docs/install/install-visual-studio.md

```diff
@@ -93,6 +93,11 @@ To install Visual Studio 2017 RC in a language of your choosing, click the **Lan
 
   > [!IMPORTANT]
   > While Visual Studio 2017 RC in general is supported for use in a production environment, those workloads and components that are marked "Preview" in the installation UI are not supported for use in a production environment.
+  
+## Changing the installer Language
+
+By default, the installer program tries to match the language of the operating system when run for the first time. The installer will remember this setting. The setting can later be chagned by running the installer from the command line. For example, the user can force the installer to run in Englishby running **vs_installer.exe --locale en-US"**. This setting will then be remembered when the installer is run the next time. 
```

Comment:
As well, I'd like to word-craft this a bit:

By default, the installer program tries to match the language of the operating system when it is run for the first time. The installer will remember this setting. You can change the setting later by running the installer from the command line. For example, you can force the installer to run in English by running the **vs_installer.exe --locale en-US"** command. This setting will then be remembered when the installer is run the next time. 

It's best to say "you" when you can, as a best practice.